packages/falter-berlin-tunnelmanager/files/up.sh

```diff
@@ -19,9 +19,81 @@ get_configured_ips() {
     ip -o -4 a s | awk -F ' |\/' '{print $7}'
 }
 
-get_ips_from_prefix "$prefix"
+cleanup_olsr_wg_config() {
+    # all_olsr_interfaces=$(echo "/int" | nc 127.0.0.1 2006 | tail +3 | awk '{print $1}')
+    olsr_wg_interfaces=$(echo "/int" | nc 127.0.0.1 2006 | grep wg_ | awk '{print $1}')
+
+    i=0
+    while uci get olsrd.@Interface[$i] &>/dev/null; do
+        int_name=$(uci get olsrd.@Interface[$i].interface)
+        if [ $? -ne 0 ]; then # check if interface config is wrong
+            uci delete olsrd.@Interface[$i]
+            continue
+        fi
+
+        # skip non wireguard interfaces
+        slicedint=$(echo $int_name | cut -c1-3)
+        if [ "${slicedint}" != "wg_" ]; then
```

Comment:
This will literally match only the interface `wg_` (wihtout a number) is that your intention?

Reply:
Ah, I see. I've only look on the if-statement. :)